<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogTrigger</class>
 <widget class="QDialog" name="DialogTrigger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>700</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>trigger text</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_add">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>730</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>send command</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>1261</width>
     <height>451</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_triggerTxt">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>700</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_sendCommandList">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>730</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_del">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>770</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>560</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>触发器使能</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_enable">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>560</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_type">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>500</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>包含</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>行首</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>行尾</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>包含变量赋值</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>500</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>触发方式</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_txt1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>540</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_txt2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>600</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_var_name">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>570</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>第一段文字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量名称</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>600</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>第二段文字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>500</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>if运行方式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_if_mode">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>500</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>布尔判段真假</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>数值运算</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>字符串相等</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>无变量直接执行</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>600</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量要操作的值</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_if_var_oprate">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>570</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_if_var_name">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>540</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>570</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量要进行的操作</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_if_var_oprate_value">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>600</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>变量名称</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cmdList">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>640</y>
     <width>711</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>if要执行的语句列表</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_else_enable">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>670</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>670</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>是否有else</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>670</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>else要执行的语句列表</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_else_cmdList">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>670</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
